### DISCUSSION TITLE
Urgent: text changes newly created events disappear after saving.

### DISCUSSION THREAD

**Discussion by JanineKall** [liedekef/events-made-easy#656] [General]
I got a message from one of my co-workers that she's not able to create a new event. First, she wrote the whole thing, than saved and it disappeared. I suggested she 'd try again, copy an old event and than saving immediately after entering the title and than editing the event. Title save was successful, but after editing the event text changes were not saved.  Website is properly updated. 
What happened? https://hb-cafe.nl 
![EME-no new event](https://github.com/user-attachments/assets/68b28e3c-1a10-4afc-a9fa-a608ba696735)

**Comment by liedekef** [comment]
Most of the time this indicates issues with the DB (db space full or db corrupted). Nothing else comes to mind in fact.
So first check the health of the DB and go from there.
